README.md

```diff
@@ -14,12 +14,18 @@ The restore service consists of a website that allows you to view your backups a
 
 ## Development
 
-### Backups
+### Jobs
 
-1. conda create --name moonwalk-backup python=3.11
-1. conda activate moonwalk-backup
-1. cd jobs/backup
-1. pip install -e ".[tests]"
+1. `conda create --name moonwalk-backup python=3.11`
```

Comment:
**question**: the firebase.json mentions 3.12 python. Should these be synchronized?